core/components/atoms/metricInput/__stories__/variants/Size.story.jsx

```diff
@@ -1,18 +1,20 @@
 import * as React from 'react';
-import { MetricInput, Text } from '@/index';
+import { MetricInput, Text, Row, Column } from '@/index';
 
 // CSF format story
 export const size = () => (
-  <div className="d-flex" style={{ width: 'var(--spacing-8)' }}>
-    <div className="d-flex flex-column align-items-center mr-8">
-      <MetricInput aria-label="Metric input regular size" size="regular" className="mb-3" />
-      <Text>Regular</Text>
-    </div>
-    <div className="d-flex flex-column align-items-center">
-      <MetricInput aria-label="Metric input large size" size="large" className="mb-3" />
-      <Text>Large</Text>
-    </div>
-  </div>
+  <Row>
+    <Column size={2} className="d-flex">
+      <div className="flex-column align-items-center mr-8">
```

Comment:
We do not require `d-flex` & `flex-column` classes when we are already using the `Row` & `Column` components

Reply:
yes we do need it or else it looks like in the image below: 
![Screenshot_2023-01-10_11-38-33](https://user-images.githubusercontent.com/99866103/211474903-9e0086e9-2044-4049-9297-e1b549fff290.png)